### PR TITLE
Correcting typo in getting_started.ipynb

### PR DESCRIPTION
**Pull request by acolombi**
Typo correction in documentation.

### CODE REVIEW

File: docs/source/notebooks/getting_started.ipynb [hunk 1]
```diff
@@ -1031,8 +1031,8 @@
     "$$ \n",
     "\\begin{aligned}  \n",
     "  D_t &\\sim \\text{Pois}(r_t), r_t= \\begin{cases} \n",
-    "   l, & \\text{if } t \\lt s \\\\\n",
-    "   e, & \\text{if } t \\ge s \n",
+    "   e, & \\text{if } t \\le s \\\\\n",
+    "   l, & \\text{if } t \\gr s \n",
```
Comment (ColCarroll) [inline docs/source/notebooks/getting_started.ipynb]:
```suggestion
    "   l, & \\text{if } t \\gt s \n",
```

**Review by ColCarroll — COMMENTED** [review]
Looks good to merge with one change- `\\gr` was not parsing - looks like the symbol was `\\gt`.

**Comment by acolombi** [comment]
whoops! thanks

**Comment by twiecki** [comment]
Thanks @acolombi!

**Pull request merged**